projections/account.go

```diff
@@ -35,11 +35,13 @@ func (h *AccountProjectionHandler) HandleEvent(
 			`INSERT INTO account (
 				id,
 				name,
-				customer_id
+				customer_id,
+				balance
```

Comment:
The `balance` column should probably default to `0` and avoid being referenced here. This is a minor form of coupling that is unnecessary.